### PR TITLE
Warn users that `forum-search` and `scratchstats` might not work

### PR DESCRIPTION
**Pull request by iqnite**
Resolves https://discord.com/channels/806602307750985799/1253390754483535945
(it doesn't actually resolve it, it makes it a known issue)

### Changes

Add a warning, like in `image-uploader`, to tell users that Forum search and ScratchStats might not work. Also add the *Beta* tag to both addons.

### Reason for changes

https://scratchdb.lefty.one

### CODE REVIEW

**Comment by Secret-chest** [comment]
Can lefty post the code of ScratchDB?

File: addons/forum-search/addon.json [hunk 1]
```diff
@@ -9,6 +9,13 @@
       "name": "Hans5958"
     }
   ],
+  "info": [
+    {
+      "id": "deprecated",
+      "type": "warning",
+      "text": "The database operating behind this addon is experiencing issues, so this addon might not work."
```
Comment (mxmou) [inline addons/forum-search/addon.json:16]:
```suggestion
      "text": "The ScratchDB database used by this addon is offline, so this addon does not work."
```

File: addons/scratchstats/addon.json [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
     {
       "name": "Maximouse",
       "link": "https://scratch.mit.edu/users/Maximouse/"
     }
   ],
+  "info": [
+    {
+      "id": "deprecated",
+      "type": "warning",
+      "text": "The database operating behind this addon is experiencing issues, so this addon might not work."
```
Comment (mxmou) [inline addons/scratchstats/addon.json:15]: This one should also be changed.

Comment (iqnite) [reply addons/scratchstats/addon.json:15]: Lol I forgot

**Review by mxmou — APPROVED** [review]

**Pull request merged**